alaoweb/2021.alaoweb.org
# Schedule

* substantially fulfills #53 
* creates schedule grid with session titles (speaker info not yet connected)
* still needs work on representing lightning talks (#54)

To test: 
See that the schedule loads
https://deploy-preview-67--keen-feynman-3ae551.netlify.app/schedule/
clicking on schedule links reveals content info (title/abstract) but mostly no speaker info yet
